### PR TITLE
Fixed #6540 - Hard coded text to language files

### LINKED ISSUE #6540: [language] Hard coded messages in Elasticsearch
An error has occurred while performing the search. Your query syntax might not be valid.
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/SearchThrowableHandler.php#L117

Unable to find the requested SearchEngine. Try performing the search again.
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/SearchThrowableHandler.php#L119

Failed to connect to the Elasticsearch server.
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/SearchThrowableHandler.php#L121

An error internal to the Search has occurred.
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/SearchThrowableHandler.php#L123

An unknown error has occurred while performing the search.
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/SearchThrowableHandler.php#L125

Contact an administrator if the problem persists. More information available in the logs.
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/SearchThrowableHandler.php#L171

ms
https://github.com/salesagility/SuiteCRM/blob/develop/lib/Search/UI/templates/search.results.tpl#L75

Search Interface
https://github.com/salesagility/SuiteCRM/blob/develop/modules/Administration/Search/view.tpl#L50

### PR DESCRIPTION
**Pull request by code-ph0y**
Fixed #6540 - Hard coded text to language files

## Description
Created language entries for the hard coded text (see Issue for more details).

## Motivation and Context
So the text can be translatable.

## How To Test This
View the text that has been changed (see Issue). Change Language file, Check that the text is changing in the appropriate sections.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: lib/Search/UI/SearchThrowableHandler.php [hunk 1]
```diff
@@ -110,19 +110,21 @@ public function handle()
...
     private function getFriendlyMessage()
     {
+        global $mod_strings;
+
         switch (get_class($this->throwable)) {
             case SearchUserFriendlyException::class:
                 return $this->throwable->getMessage();
             case SearchInvalidRequestException::class:
-                return 'An error has occurred while performing the search. Your query syntax might not be valid.';
+                return $mod_strings['LBL_ELASTIC_SEARCH_SEARCH_INVALID_REQUEST'];
```
Comment (Dillon-Brown) [inline lib/Search/UI/SearchThrowableHandler.php]: Do these appear to users in the CRM?

File: modules/Administration/language/en_us.lang.php [hunk 2]
```diff
@@ -968,6 +968,7 @@
     'LBL_ELASTIC_SEARCH_INDEX_SCHEDULE_PART_FAIL' => 'Failed to start partial index.',
 
     // SearchWrapper Settings
+    'LBL_SEARCH_INTERFACE' => 'Search Interface',
     'LBL_SEARCH_ENGINE' => 'Search Engine',
     'LBL_SEARCH_ENGINE_HELP' => 'Configure the default search engine used by the search wrapper',
```
Comment (Dillon-Brown) [inline modules/Administration/language/en_us.lang.php]: Might want to update this while your here: https://github.com/salesagility/SuiteCRM/issues/6540#issuecomment-445213085

File: lib/Search/UI/SearchThrowableHandler.php (hunk 1 above)
Comment (code-ph0y) [reply lib/Search/UI/SearchThrowableHandler.php]: Yes.

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Comment by likhobory** [comment]
See also PR #6841

File: modules/Home/language/en_us.lang.php [hunk 3]
```diff
@@ -238,4 +238,11 @@
     'LBL_STATUS' => 'Status: ',
     'LBL_DESCRIPTION' => 'Description: ',
     //End Events
+
+    'LBL_ELASTIC_SEARCH_EXCEPTION_SEARCH_INVALID_REQUEST' => 'An error has occurred while performing the search. Your query syntax might not be valid.',
+    'LBL_ELASTIC_SEARCH_EXCEPTION_SEARCH_ENGINE_NOT_FOUND' => 'Unable to find the requested SearchEngine. Try performing the search again.',
```
Comment (horus68) [inline modules/Home/language/en_us.lang.php]: SearchEngine should be "Search Engine"

**Pull request merged**